proto/storage/vuln_requests.proto

```diff
@@ -31,12 +31,26 @@ enum RequestStatus {
 }
 
 message RequestExpiry {
+    enum ExpiryType {
+        TIME            = 0;
```

Comment:
Should we have an unset here for compatibility purposes? Or is Time essentially the default?